app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -12,6 +12,7 @@ class Insured::ConsumerRolesController < ApplicationController
   before_action :set_cache_headers, only: [:edit, :help_paying_coverage, :privacy, :search]
   before_action :redirect_if_medicaid_tax_credits_link_is_disabled, only: [:privacy, :search]
   before_action :sanitize_contact_method, only: [:update]
```

Comment:
Can you replace this with below version.
```suggestion
  before_action :sanitize_contact_method, :validate_person_match, only: [:update]
```